bio.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bio</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <header>
+        <a href="index.html"><h1 class="logoText">TS</h1></a>
+        <nav class="headerNav"> 
+            <ul>
+                <li> <a  href="portfolio.html">Portfolio</a></li>  
+                <li><a href="experience.html">Experience</a></li>
+                <li> <a class = "active" href="bio.html">Bio</a></li>
```

Comment:
nitpick: make sure that the spacing before and after the equals in the HTML attributes are consistent throughout the files